workflows/mag.nf

```diff
@@ -667,8 +668,19 @@ workflow MAG {
             )
             ch_versions = ch_versions.mix(PROKKA.out.versions.first())
         }
+
+        /*
+        * DAS_Tool: binning refinement
+        */
+        if ( params.refine_bins_dastool && !params.skip_metabat2 && !params.skip_maxbin2 ) {
+            BINNING_REFINEMENT ( BINNING_PREPARATION.out.grouped_mappings, BINNING.out.bins )
+            ch_versions = ch_versions.mix(BINNING_REFINEMENT.out.versions)
```

Comment:
not sure, since you added all these ".first()"s shouldn't be here be one as well?
```suggestion
            ch_versions = ch_versions.mix(BINNING_REFINEMENT.out.versions.first())
```

Reply:
not in this case, as BINNING_REFINEMENT is a subworkflow so exporting multiple `versions.yml` from the various tools inside.